### DISCUSSION TITLE
Pasting clipboard contents does not work in source code mode

### DISCUSSION THREAD

**Discussion by rokka** [cefsharp/CefSharp#4588] [Q&A]
Hi,

we are using CefSharp 101.0.18 in WinForms and have the problem that pasting the clipboard content with the shortcur Strg+V does not work in source code mode.

Any ideas on this?

Greatings
Andre

**Comment by amaitland** [comment]
> in source code mode.

What is `source code mode` exactly?

How does the current supported version behave? (`M116` at time of writing)
